packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -2721,18 +2716,12 @@ export class SnapController extends BaseController<
         return existingSnap;
       }
 
-      return await this.updateSnap(
+      return await this.#updateSnap({
         origin,
         snapId,
         location,
         versionRange,
-        // Since we are requesting an update from within processRequestedSnap,
-        // we disable the emitting of the snapUpdated event and rely on the caller
-        // to publish this event after the update is complete.
-        // This is necessary as installSnaps may be installing multiple snaps
-        // and we don't want to emit events prematurely.
```

Comment:
Is this no longer relevant?

Reply:
Seemed to reference dead code, it was always called with `false` internally which doesn't emit the event.